frontend/src/util/RedactionState.js

```diff
@@ -0,0 +1,11 @@
+import { atom, atomFamily } from "recoil";
```

Comment:
If you don't need this file for the rest of the PR, maybe split it off into the redaction PR? Like don't include it in this one